core/src/main/java/com/linecorp/armeria/common/RequestContext.java

```diff
@@ -350,6 +350,12 @@ default ExecutorService makeContextAware(ExecutorService executor) {
     }
 
     /**
+     * Returns whether this {@link RequestContext} has been timed-out (e.g., when the corresponding request
+     * passes a deadline).
+     */
+    boolean timedOut();
```

Comment:
Which do you usually use? I know armeria doesn't use java-bean naming (get/set) so not sure what the style is for booleans. Either is fine with me